### ISSUE TITLE
Docker image v0.29.1 disappeared on docker hub

### ISSUE THREAD

**Issue by JonasPf** [certbot/certbot#6935]
We use the docker image with the tag v0.29.1 from https://hub.docker.com/r/certbot/certbot/. It looks like that image disappeared.

I like to pin to a specific version to have consistent behaviour. Is there a policy in place for deprecating/removing certain versions? What is the best way for me to ensure that the version I'm using still exists?

**Comment by bmw** [comment]
This was my fault. Our setup for automatically updating Docker Hub was broken (see #6630) and I was doing it manually yesterday. When I went to push to the Certbot repo, I forgot to include the tag and it pushed all tags I had for that image locally. This included the v0.29.1 image.

Not being sure if my changes to that tag were correct, I just deleted it, but I'll investigate and reupload the tag. In general, we have no plans to remove tags like this and thanks for flagging the issue so I can fix it properly.
